app/src/androidTest/java/end2end/MainToASLRecognitionTest.kt

```diff
@@ -52,7 +52,8 @@ class MainToASLRecognitionTest {
     composeTestRule.onNodeWithTag("CameraFeedbackButton").assertIsDisplayed().performClick()
 
     // Go to ASLRecognition screen
-    composeTestRule.onNodeWithTag("aslRecognitionTitle").assertIsDisplayed()
```

Comment:
It seems you replaced the screen title with a popup, which is much clearer and more informative—great job!